iocore/net/SSLNetVConnection.cc

```diff
@@ -943,7 +942,7 @@ SSLNetVConnection::free(EThread *t)
 int
 SSLNetVConnection::sslStartHandShake(int event, int &err)
 {
-  if (stop_ssl_handshake) {
+  if (TsStatus::is_ssl_handshaking_stopped()) {
```

Comment:
Looks like I forgot to add the `unlikely` here in my pr, could you add that?

Reply:
Looks like it works to put likely/unlikely inside an inline function:  https://godbolt.org/z/EjwZrJ